src/K8sJanitor.WebApi/EventHandlers/CapabilityRegisteredEventHandler.cs

```diff
@@ -40,7 +40,7 @@ public async Task HandleAsync(CapabilityRegisteredDomainEvent capabilityRegister
 
             await _namespaceRepository.CreateNamespaceAsync(
                 namespaceName: namespaceName,
-                roleName: configmapRoleName
+                accountId: configmapRoleName
```

Comment:
Is configmapRoleName the value that is actually expected here, after the change to accountId from roleName?

The configmapRoleName seem to stem from:
```
            var capabilityName = capabilityRegisteredDomainEvent.Payload.CapabilityName.ToLower();
            
            var configmapRoleName = capabilityName;
```

But is used in _CreateNamespaceAsync_ as a "accountId", which eventually ends up being used in a call to _IAM.ConstructRoleArn_.


That function is a one liner that looks like this:
`return $"arn:aws:iam::{accountId}:role/{roleName}";`


_CreateNamespaceAsync_ Gives it the "configmapRoleName" value as "accountId" and a string literal with the value of "*" as roleName, which would eventually end up looking like
`arn:aws:iam::HelloPelle:role/*`

Now, I'm not too versed with the IAM format, but would that work? A quick look at the [docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html) seem to indicate that the accountId is a number,but the capabilityName is quite likely a combination of letters and numbers, and presumably can't be used interchangeably in this case.

Reply:
Fixed